arpwatch.te

```diff
@@ -91,6 +93,8 @@ userdom_dontaudit_use_unpriv_user_fds(arpwatch_t)
 
 mta_send_mail(arpwatch_t)
 
+filetrans_pattern(system_mail_t, arpwatch_data_t, mail_home_rw_t, dir, ".esmtp_queue")
```

Comment:
For this named_filetrans interface should be used or created if doesn't exists. For more info see:https://github.com/fedora-selinux/selinux-policy-contrib/blob/rawhide/mta.if#L1292